src/netcdf_writer.jl

```diff
@@ -145,10 +164,13 @@ function NetCDFWriter(
         ClimaComms.iamroot(comms_ctx) ?
         Dict{String, ClimaComms.array_type(space)}() : Dict{String, Nothing}()
 
+    unsynced_datasets = Set{String}()
```

Comment:
```suggestion
    unsynced_datasets = Set{NCDatasets.NCDataset}()
```
Won't this need to be a different type?

Reply:
Thank you!

Now I am wondering how the tests have passed with this...

Reply:
🤔